workspace.json

```diff
@@ -2569,7 +2569,7 @@
               },
               {
                 "glob": "*",
-                "input": "apps/judicial-system/api/pdf-templates",
+                "input": "apps/judicial-system/api/templates",
```

Comment:
This tripped me up in the nx10 PR, which is more strict :)

Reply:
Yes, I noticed this when I was viewing your branch and fixed id :)